vespa_jersey2/pom.xml

```diff
@@ -46,6 +46,10 @@
       <groupId>com.fasterxml.jackson.datatype</groupId>
       <artifactId>jackson-datatype-jdk8</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Please add a comment to replace this with jackson-modules-java8 from jackson 2.8.5